core/src/main/scala/org/ensime/config/EnsimeConfigProtocol.scala

```diff
@@ -28,17 +27,18 @@ object EnsimeConfigProtocol {
   private implicit val moduleFormat: SexpFormat[EnsimeModule] = cachedImplicit
   private implicit val configFormat: SexpFormat[EnsimeConfig] = cachedImplicit
 
-  def parse(config: String): EnsimeConfig = {
+  def parse(config: String, serverConfig: String): EnsimeConfig = {
```

Comment:
why would you want to parse both files at the same time?

That doesn't matter. There is no need to parse anything for the serverconfig at this time, the construction of the object should be done by reading system properties at this stage, e.g. in a convenience method in `util`. After that works, we can look at creating a HOCON parser (**not** an s-expression parser).